.agents/skills/react-native-best-practices/references/js-animations-reanimated.md

```
@@ -0,0 +1,310 @@
+```
+
+Add to `babel.config.js`:
+
+```javascript
+module.exports = {
+	plugins: ['react-native-worklets/plugin'], // Must be last
+}
+```
+
+> **Note**: Reanimated 4 requires React Native's **New Architecture** (Fabric + TurboModules). The Legacy Architecture is no longer supported. If upgrading from v3, see the migration notes at the end of this document.
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)